src/deepsparse/v2/pipeline.py

```diff
@@ -50,39 +53,41 @@ def __init__(
         ops: Union[Dict[str, Operator], List[Operator]],
         router: Router,
         schedulers: List[OperatorScheduler],
+        continuous_batching_scheduler: ContinuousBatchingScheduler,
         pipeline_state: PipelineState = None,
     ):
 
         self.ops = ops
         self.router = router
         self.schedulers = schedulers
         self.pipeline_state = pipeline_state
+        self._continuous_batching_scheduler = continuous_batching_scheduler
         self.validate()
 
         self._scheduler_group = SchedulerGroup(self.schedulers)
 
-    def _run_sequential(
```

Comment:
I'd leave a bit more comments on what's going on. A developer who takes a peek into this functionality will probably have a lot of questions, regardless of her domain knowledge. The fact that we create new engines on demand is not easy to capture from the code alone (and why we do it?)

Reply:
nothing here is creating a new engine. that would have to be defined within the `continuous_batching_scheduler`. This is just scheduling.